File: OliteBackend/src/main/java/com/olite/app/dto/PedidoDTO.java
package com.olite.app.dto;

import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.List;

@Data
public class PedidoDTO {
    private Integer idPedido;
    private LocalDateTime fechaPedido;
    private BigDecimal total;
    private String estadoPedido;
    private String metodoPago;
    private List<PedidoDetalleDTO> detalles;
}

File: OliteAndroid/app/src/main/java/com/olite/app/model/dto/PedidoDTO.kt
package com.olite.app.model.dto

data class PedidoDTO(
    val idPedido: Int,
    val fechaPedido: String,
    val total: Double,
    val estadoPedido: String,
    val metodoPago: String,
    val detalles: List<PedidoDetalleDTO>
)